notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -122,7 +122,7 @@
       "source": [
```

Comment:
<p>Add text cell</p><p> </p><p>The import should be moved up into the prev cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>added and updated...</p>